lld/ELF/InputSection.cpp

```diff
@@ -309,6 +310,12 @@ std::string InputSectionBase::getObjMsg(uint64_t off) const {
       .str();
 }
 
+PotentialSpillSection::PotentialSpillSection(InputSectionBase *source,
```

Comment:
We prefer `const &`  when a parameter is non-null.

Reply:
Done.